updates/31/asset_collections_updates.sql

```diff
@@ -1,2 +1,4 @@
 INSERT INTO asset_collections(id, name, symbol) VALUES (342, "Morpho", "MORPHO");
 *
+INSERT INTO asset_collections(id, name, symbol) VALUES (342, "Morpho", "MORPHO");
```

Comment:
what are you doing here? I guess this is a typo? You are just duplicating the old one and my PR did not create a new collection. I will remove this.